tf2onnx/function/matrixbandpart.py

```diff
@@ -0,0 +1,82 @@
+"""
+tf2onnx.tf2onnx - matrixbandpart op conversion
+"""
+import numpy as np
+from onnx import helper, onnx_pb
+from tf2onnx import utils
+from tf2onnx.utils import make_sure
+
+
+# pylint: disable=useless-return,broad-except,logging-not-lazy,unused-argument,missing-docstring
+
+def matrixbandpart_op(ctx, node, name, args):
+    # T output = matrix_band_part(T input, int num_lower, int num_upper)
+    # data-flow: first generate mask matrix and then use element-wise mul op
+    input_rank = ctx.get_shape(node.input[0])
+    make_sure(input_rank != 2, error_msg="MatrixBandPart op: only rank2 is supported")
```

Comment:
it should be "input_rank == 2", right?

Reply:
yes.
my mistake, when refactoring the code, i forget to call function "len" to get input_rank...